resources/users/users.route.js

```diff
@@ -1,9 +1,11 @@
-const express = require('express');
+const express = require("express");
 const router = express.Router();
-const controller = require('./users.controller');
```

Comment:
Please use a good naming convention why not called `controller` ` userController`